apps/condo/domains/marketplace/access/MarketPriceScope.js

```diff
@@ -0,0 +1,108 @@
+/**
+ * Generated by `createschema marketplace.MarketPriceScope 'itemPrice:Relationship:MarketItemPrice:CASCADE; property:Relationship:Property:CASCADE; organization:Relationship:Organization:CASCADE;'`
+ */
+
+const { uniq, compact, get } = require('lodash')
+
+const { throwAuthenticationError } = require('@open-condo/keystone/apolloErrorFormatter')
+const { find, getById } = require('@open-condo/keystone/schema')
+
+const { queryOrganizationEmployeeFor, queryOrganizationEmployeeFromRelatedOrganizationFor } = require('@condo/domains/organization/utils/accessSchema')
+const { checkPermissionInUserOrganizationOrRelatedOrganization } = require('@condo/domains/organization/utils/accessSchema')
+const { RESIDENT } = require('@condo/domains/user/constants/common')
+
+async function canReadMarketPriceScopes ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
+    if (user.isAdmin || user.isSupport) return {}
+
+    if (user.type === RESIDENT) {
+        const userResidents = await find('Resident', { user: { id: user.id }, deletedAt: null })
+        if (!userResidents.length) return false
+        const residentOrganizationsIds = compact(userResidents.map(resident => get(resident, 'organization')))
+        const residentsIds = userResidents.map(resident => resident.id)
+        const userServiceConsumers = await find('ServiceConsumer', {
```

Comment:
MarkerPriceScope shows which property can see the price. So, we don't need ServiceConsumers here. Just need to filter by resident.property and organization.
```js
{
  OR: [
    // Scopes with a particular property
    { property: { id_in: residents.map(({property}) => property) } },
    // or organization's default scopes
    {
      AND: [
        { property_is_null: true },
        { marketItemPrice: { marketItem: { organization: { id_in: resident.map(({organization}) => organization) } } } },
      ],
    },
  ],
}
```